### PR TITLE
Add eval score

### PR DESCRIPTION
**Pull request by alalulu8668**
add eval-score agent to evaluate llm-based agent.

### CODE REVIEW

File: tests/test_eval_agent.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    reference_answer_list = list(query_answer['ChatGPT (3.5) Answer'])
+    chatgpt_answer_list = list(query_answer['ChatGPT Direct Answer'])
+    BMZ_chatbot_answer_list = list(query_answer['BMZ Chatbot Answer'])
+    
+    
+    for i in range(len(question_list)):
+        question = question_list[i]
+        reference_answer = reference_answer_list[i]
+        gpt_direct_answer = chatgpt_answer_list[i]
+        chatbot_answr = BMZ_chatbot_answer_list[i]
```
Comment (oeway) [inline tests/test_eval_agent.py:76]: typo

File: tests/test_eval_agent.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            constraints=None,
+            actions=[bot_answer_evaluate],
+            model="gpt-4"
+        )
+    event_bus = evalBot.get_event_bus()
+    event_bus.register_default_events()
+    
+    query_answer = load_query_answer()
+    question_list = list(query_answer['Question'])
+    reference_answer_list = list(query_answer['ChatGPT (3.5) Answer'])
```
Comment (oeway) [inline tests/test_eval_agent.py:67]: Update title name to "with context " and download the csv file again.

**Comment by alalulu8668** [comment]
Screenshot 2023-11-27 at 11 10 25 AM

**Comment by alalulu8668** [comment]
image

**Comment by alalulu8668** [comment]
Metric description:
image

**Pull request merged**